### DISCUSSION TITLE
Badge Scanner Hardware and Software used at GitHub Universe 2023

### DISCUSSION THREAD

**Discussion by knxp34** [community/community#77583] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello

At Universe 2023 there were custom coded badges that used the hardware pictured.  I would love to know more about the hardware and software used for this.  I hear the code was written by Copilot, but would love to know more about it 👍🏻 

![programable_badges_withcopilot](https://github.com/community/community/assets/70229485/b6d8dcea-3700-4a86-8ec0-c591a3e0eee9)

Thanks
-Tom Printy

**Comment by queenofcorgis** [comment]
Hey Tom! Glad you liked the badges: here is the user guide which should answer your software and hardware questions: https://learn.pimoroni.com/article/getting-started-with-badger-2040.

**Reply by queenofcorgis** [reply]
Also more info in the Badger Repo: https://github.com/badger2040/home which is pretty similar to the above.

**Comment by knxp34** [comment]
Thank you so much!  This is pretty great and pretty cool :)

**Comment by knxp34** [comment]
Also next time you need barcode scanners please reach out to me. Would love to Get some Zebra Barcode scanners in on these projects like this.
